packages/status-js/src/utils/deserialize-public-key.ts

```diff
@@ -0,0 +1,73 @@
+import { Point } from 'ethereum-cryptography/secp256k1'
+import {
+  toHex,
+  // utf8ToBytes as toBytes, // see https://github.com/paulmillr/noble-hashes/blob/d76eb7c818931d290c4c27abb778e8e269895154/src/utils.ts#L91-L96
```

Comment:
ℹ️ Could be removed, but decided to leave if ever deciding to uses bytes from the start and wondering where the `toBytes` function is (since it's not exported).